### ISSUE TITLE
Feature: Feature: CLI to line number

### ISSUE THREAD

**Issue by mjurczyk** [slap-editor/slap#168]
## Init `slap` on a specific line in a file

While working today with `slap` I have noticed that it might be exceptionally useful to be able to open file at an exact line/char with slap. Node and npm usually deliver error messages in a form:

```
stack: What a Terrible Failure: I told you undefined wasn't a function ...
   at /Users/mjurczyk/Project/index.js:220:68
   at processImmediate [as _immediateCallback] (timers.js:371:17)
```

As a user I would love to be able to just copy the file path from the error message, paste it into terminal and slap the bug in the face.

### Usage
To open a file on line 21 and character 2:
`slap ./file.js:21:2`

Character parameter is optional:
`slap ./file.js:108`

### Behaviour

* Default cursor position is 1, 1 in top left corner
* Clip cursor to the file content
* Reset selection's tail
* Skip for empty/new files

PR https://github.com/slap-editor/slap/pull/167 (testable on my fork on `npm install -g mjurczyk/slap#feature-cli-at-line`)

**Comment by rahsd** [comment]
:+1:

**Comment by jczimm** [comment]
+1

**Comment by dbkaplun** [comment]
This has been fixed as of [dbkaplun/editor-widget@f4cc322](https://github.com/dbkaplun/editor-widget/commit/f4cc322cd85a0c97de7af71449ccbe2d7f5d5cf2) and slap@0.1.43. Let me know if you have problems and I'll reopen!

**Comment by fth86** [comment]
Opening a file via  slap file:line:column doesnt work in the current build (slap@0.1.60).

strace reports an error in  .npm-packages/lib/node_modules/slap/lib/ui/Slap.js:67:14,  a line that reads:

 if (fs.lstatSync(filePath).isDirectory()) throw _.merge(new Error('EISDIR: illegal operation on a directory, read'), {cause: {code: 'EISDIR'}});

dunno if this is the real cause of it not working, but could be a clue, ...
